rbmc-cfam-daemon/src/cfam_tool.cpp

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+#include "local_cfam.hpp"
+
+#include <CLI/CLI.hpp>
+
+#include <format>
+#include <iostream>
+#include <string_view>
+
+void displayCFAMFields(std::string_view name, size_t link)
+{
+    SysFSImpl sysfs;
+    LocalCFAM cfam{link, sysfs};
+
+    std::cout << std::format("{} CFAM scratchpad fields\n", name);
+
+    for (const auto& [field, meta] : LocalCFAM::cfamFields)
+    {
+        auto data = cfam.readField(field);
+        if (!data.has_value())
+        {
+            std::cerr << std::format("Unable to read {} CFAM ({})\n", name,
+                                     data.error());
+            return;
+        }
+
+        std::cout << std::format("{:24} {:#x}\n", meta.desc, data.value());
```

Comment:
It would be helpful if we printed the value in a more readable format.

For example,
```
Local BMC CFAM scratchpad fields
API Version              0x1
BMC Position             0x0
Role                     Active/Passive
Redundancy Enabled       true/false
Failovers Paused         true/false
Provisioned              true/false
BMC State                Ready/(BMC state enumeration name) 
Sibling Communication OK true/false
Heartbeat                0x3b
FW Version               0x1c9c4045

Reply:
I was afraid you were going to say that.

How would I accomplish that in a generic way in the code?

Reply:
Can't say it's pretty but I did the requested formatting.